crates/asm/subprotocols/bridge-v1/src/errors.rs

```diff
@@ -105,3 +105,23 @@ pub enum WithdrawalCommandError {
     #[error("Deposit amount mismatch {0}")]
     DepositWithdrawalAmountMismatch(Mismatch<u64>),
 }
+
+/// Error type for OperatorBitmap operations.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum BitmapError {
+    /// Attempted to set a bit at an index that would create a gap in the bitmap.
+    /// Only sequential indices are allowed.
+    IndexOutOfBounds(OperatorIdx),
+}
+
+impl std::fmt::Display for BitmapError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            BitmapError::IndexOutOfBounds(idx) => {
+                write!(f, "Index {} is out of bounds for sequential bitmap", idx)
+            }
+        }
+    }
+}
+
+impl std::error::Error for BitmapError {}
```

Comment:
Why not just add `thiserror` to the `Cargo.toml` and add the `Error` to the derive?
Less code to maintain.

Reply:
Actually it's already there!
So, no excuses 😂